smdebug/tensorflow/keras.py

```diff
@@ -392,7 +409,7 @@ def _add_callbacks(self, mode):
         # self.original_fetches = self._get_exec_function(mode).fetches.copy()
 
         x = self._get_exec_function(mode)  # Returns GraphExecutionFunction
-        if self._validate_exec_function(x):
+        if x and self._validate_exec_function(x):
```

Comment:
why is this needed? comment line 376 needs to be here?

Reply:
can be removed

Reply:
removed in latest commit